persistence/src/test/java/cz/muni/fi/pa165/sportsclub/dao/CoachDaoImplTest.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.pa165.sportsclub.dao;
+
+import cz.muni.fi.pa165.sportsclub.PersistenceSampleApplicationContext;
+import cz.muni.fi.pa165.sportsclub.entity.Coach;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Transactional;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+import javax.validation.ConstraintViolationException;
+
+
+/**
+ * Tests for CoachDaoImpl
+ *
+ * @author 410461 Martin Skrovina
+ */
+@ContextConfiguration(classes = PersistenceSampleApplicationContext.class)
+@TestExecutionListeners(TransactionalTestExecutionListener.class)
+@Transactional
+public class CoachDaoImplTest extends AbstractTestNGSpringContextTests {
+
+    @Inject
+    private CoachDao coachDao;
+
+    @Test
+    public void testCreate() {
+        Coach coach = new Coach();
+        coach.setFirstName("Firstname");
+        coach.setLastName("Lastname");
+        coach.setEmail("lastname@example.com");
+
+        coachDao.create(coach);
+
+        Assert.assertEquals(coachDao.findCoachById(coach.getId()), coach);
```

Comment:
I think the order should be expected, actual

Reply:
Please explain what you mean, @HonzaCech

Reply:
In assertEquals(), the order of the parameters should be expected, actual. In case a test fails, it writes something like "Expected value x, actual value y", so it would be confusing to have it reversed.

Reply:
Sorry, I didn't realise we were using different assert framework, your code is good